Storage/Bookmarks/Bookmarks.swift

```diff
@@ -14,6 +14,10 @@ public protocol SearchableBookmarks: class {
     func bookmarksByURL(_ url: URL) -> Deferred<Maybe<Cursor<BookmarkItem>>>
 }
 
+public protocol CountableBookmarks: class {
+    func numberOfBookmarks() -> Deferred<Maybe<Int>>
```

Comment:
Remember that `ViewBookmarksLocalOnMirror` is only local and fully synchronized bookmarks, but in the regular bookmarks view we also show the buffer if no bookmarks have been merged. For illustration, in my account right now `VBLOM` has about 15 bookmarks, while in the bookmarks panel I have access to about 3,000.

If you don't mind the potential for slight over-counting, we can fairly easily get an estimate of all non-folder items, and I'm happy to help with the SQL.